src/pages/AboutUs/Clients.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
this file is a component so please move it into `src/component` folder

Reply:
aw well as the others components

Reply:
Oh I think you only forget to remove this file cuz there is already another one in the component folder 
so you just have to move the the photos to shared file and call it something like `media` or any meaningful name